assets/js/lib/test-utils/factories/clusters.js

```diff
@@ -54,6 +54,7 @@ export const hanaClusterDetailsFactory = Factory.define(() => ({
   stopped_resources: clusterResourceFactory.buildList(2),
   system_replication_mode: 'sync',
   system_replication_operation_mode: 'logreplay',
+  maintenance_state: false,
```

Comment:
can we name this `maintenance_mode` to keep the same value as in the backend?
most probably the tests are passing by coincidence hehe

Reply:
done!